flytekit/core/node.py

```diff
@@ -47,6 +47,8 @@ class Node(object):
     ID, which from the registration step
     """
 
+    timeout_override_sentinel = object()
```

Comment:
Nit: I usually see python sentinels as constants, thus all capitalized: 

```suggestion
    TIMEOUT_OVERRIDE_SENTINEL = object()
```

Reply:
sounds good - just updated